From: Simon McVittie <smcv@debian.org>
Date: Wed, 24 Jan 2024 13:55:12 +0000
Subject: tests: Skip composefs test if /var/tmp does not support user xattrs

Otherwise, this test fails on Debian 12 (Linux 6.1) kernels if /var/tmp
is a tmpfs. Some autobuilders put the entire build chroot on a tmpfs,
to speed up builds.

Signed-off-by: Simon McVittie <smcv@debian.org>
Forwarded: https://github.com/ostreedev/ostree/pull/3145
---
 tests/test-composefs.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/test-composefs.sh b/tests/test-composefs.sh
index 4b91973..d05579f 100755
--- a/tests/test-composefs.sh
+++ b/tests/test-composefs.sh
@@ -24,6 +24,7 @@ if ! ${CMD_PREFIX} ostree --version | grep -q -e '- composefs'; then
     exit 0
 fi
 
+skip_without_user_xattrs
 
 setup_test_repository "bare-user"
 
